SubmittedAssignments/TS-view-invoice-format-1/index.html

```
@@ -0,0 +1,105 @@
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    
+        <invoice-component 
```

HTMLHint (1.5.0)

[error] The value of attribute [ config ] must be in double quotes.

(attr-value-double-quotes)